.eslintrc.json

```diff
@@ -28,6 +28,7 @@
     "sort-imports": [
       "error",
       { "ignoreDeclarationSort": true, "ignoreCase": true }
-    ]
+    ],
+    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
```

Comment:
Don't warn on unsued variables that start with `_`, e.g.: https://github.com/confio/ts-relayer/pull/134/files#diff-e8c0ad1a88c4869847d33b7631f1c6a11037972cd3ac65ecab3834ef0f0bf617R25